src/cloud-api-adaptor/podvm/qcow2/build-s390x-se-image.sh

```diff
@@ -190,12 +206,18 @@ echo "Generating an IBM Secure Execution image"
 echo "Creating SE boot image"
 export SE_PARMLINE="root=/dev/mapper/$LUKS_NAME rd.auto=1 rd.retry=30 console=ttysclp0 quiet panic=0 rd.shell=0 blacklist=virtio_rng swiotlb=262144"
 sudo -E bash -c 'echo "${SE_PARMLINE}" > ${dst_mnt}/boot/parmfile'
+if [ "${SE_VERIFY}" = "true" ]; then
+    EXTRA_ARGS = " --cert=${cacert} --cert=${signcert} --crl=${crl} "
+else
+    EXTRA_ARGS = " --no-verify "
```

Comment:
I don't think assignment works with spaces around the `=`
```suggestion
    EXTRA_ARGS=" --cert=${cacert} --cert=${signcert} --crl=${crl} "
else
    EXTRA_ARGS=" --no-verify "
```

Reply:
I've modified and tested
```
[root@bastion-rr-initdata ~]# oc get pods
NAME          READY   STATUS    RESTARTS   AGE
busybox-new   1/1     Running   0          54s
```